src/mono/mono/mini/interp/interp.c

```diff
@@ -8611,4 +8605,59 @@ mono_interp_is_method_multicastdelegate_invoke (MonoMethod *method)
 {
 	return is_method_multicastdelegate_invoke (method);
 }
+
+// after interp_entry_prologue the wrapper will set up all the argument values
+//  in the correct place and compute the stack offset, then it passes that in to this
+//  function in order to actually enter the interpreter and process the return value
+EMSCRIPTEN_KEEPALIVE void
+mono_jiterp_interp_entry (JiterpEntryData *_data, stackval *sp_args, void *res)
```

Comment:
Another option here would have been to make `interp_exec_method` static again as it was before the original jiterpreter commit and then have a global `mono_interp_exec_method` that calls it.

Reply:
Do you want me to change it? I originally had a wrapper like you describe, but the performance impact was measurable so that's how we ended up here. I might be able to get it to inline though, I don't think I spent much time trying when I was first prototyping.